src/components/form/FormSectionHeader.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { css } from 'glamor';
+
+export default function FormSectionHeader({ className, children, ...props }) {
+  return (
+    <div
+      {...css({
+        fontWeight: 'bold',
+        display: 'flex',
+        justifyContent: 'space-around',
+        alignItems: 'center',
+        marginBottom: '0.5rem',
+        '&.new-form-section-header-types': {
+          alignItems: 'flex-start',
+          justifyContent: 'space-around',
+        },
+        '& > *': { marginRight: '1rem' },
+        '& > *:last-child': { marginRight: '0' },
+        '& .new-form-section-header-svg': {
+          width: '1.25rem',
+          height: '1.25rem',
+          flex: '0 0 1.25rem',
+        },
+        '& .new-form-date-time-input': { flex: '1 1 auto' },
```

Comment:
This nesting will need to be changed too, obviously, since these CSS classes will be going away. This can be done later though 😄

Reply:
Do you mean removing all the style-related class names?

Reply:
Yeah. I think this component should set its own styles, but not those of descendants.

For example, in dac04b0, I removed the nested `new-form-rating-radio-label` style and moved it into its own component.

Reply:
I see what you mean. 🤔  I'll see if I can take a similar approach to refactor a little bit.